src/sentry/static/sentry/app/views/admin/adminProjects.tsx

```diff
@@ -1,32 +1,52 @@
-/* eslint-disable react/jsx-key */
 import React from 'react';
+import {RouteComponentProps} from 'react-router';
 import moment from 'moment';
 
 import ResultGrid from 'app/components/resultGrid';
 import {t} from 'app/locale';
+import {Organization} from 'app/types';
+import AsyncView from 'app/views/asyncView';
 
-export const prettyDate = function (x) {
-  return moment(x).format('ll');
+const prettyDate = (x: string) => moment(x).format('ll');
+
+type Row = {
+  dateCreated: string;
+  name: string;
+  organization: Organization;
+  slug: string;
```

Comment:
is this a projectSlug? if yes, should we update it making things more clear?

Reply:
It is but it's clear with the context

Reply:
It is also serialized as slug, not changing the name of the column in the api